components/shared/sheet/LoginSheet.tsx

```diff
@@ -27,7 +28,10 @@ function LoginSheet({ open }: { open: boolean }) {
                 Log ind med UNI•Login
               </div>
               <div>
-                <LoginButton url="/auth/login/unilogin" />
+                <LoginButton
+                  url="/auth/login/unilogin"
```

Comment:
Not that you have to implemented it right now, but this route should be defined like our other internal routes in [routes.ts](https://github.com/danskernesdigitalebibliotek/dpl-go/blob/d837bc3f60ad33bb174ca3a497eb30c5a3abfb91/lib/config/resolvers/routes.ts#L1-L12).
Consider fixing it right a way or write a TODO otherwise